File: Ejercicios Propuestos 1/Ejercicio 13/EJERCICIO_13.cpp
#include <iostream>
using namespace std;
int main() {
    long long suma = 0;       
    long double producto = 1;   
    for (int i = 20; i <= 400; i += 2) {
        suma += i;        
        producto *= i;    
    }

    cout << "La suma de números pares entre 20 y 400 es: " << suma << endl;
    cout << "El producto de números pares entre 20 y 400 es: " << producto << endl;

    return 0;
}

File: Ejercicios Propuestos 2/Ejercicio 13/Ejercicio 13.py
NAME = "Renzo Leonardo Cienfuegos Cardenas"
GROUP = 2

# Creamos dos variables acumuladoras
sumaPar = 0
sumaImp = 0

inicio = int(input("Ingrese el número de INICIO: "))
final = int(input("Ingrese el número de FIN: "))

print("-" * 50)

for i in range(inicio, final + 1):

    # Verificar si el número es par o impar
    if i % 2 == 0:
        # Si el residuo es 0, es PAR
        sumaPar += i
    else:
        # Si el residuo no es 0, es IMPAR
        sumaImp += i

# Mostrar resultados
print(f"Rango: del {inicio} al {final}")
print(f"-> La suma total de los números PARES es: {sumaPar}")
print(f"-> La suma total de los números IMPARES es: {sumaImp}")
print("-" * 50)
